airflow/www/views.py

```diff
@@ -835,7 +835,9 @@ def index(self):
 
             is_paused_count = dict(
                 session.execute(
-                    select(DagModel.is_paused, func.count(DagModel.dag_id)).group_by(DagModel.is_paused)
+                    all_dags.with_only_columns([DagModel.is_paused, func.count()])
+                    .where(DagModel.is_active)
```

Comment:
I don't think we need the `DagModel.is_active` clause. It's already present in the `all_dags`/`dags_query`

Reply:
You're right. I just removed the redundant where clause.